.github/workflows/DeployStaticExecutables.yaml

```diff
@@ -0,0 +1,106 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+name: Deploy static executables
+
+on:
+  release:
+    types: [published]
```

Comment:
Could you switch this to:

```yaml
  workflow_call:
    inputs:
      tag_name: ... # same as below
    secrets:
      DOCKERHUB_USERNAME:
        required: true
      DOCKERHUB_TOKEN:
        required: true
```

Then at the bottom of `PostRelease.yaml` add:

```yaml
  static_execs:
    uses: ./.github/workflows/DeployStaticExecutables.yaml
    secrets: inherit
    with:
      tag_name: ${{ inputs.tag_name || github.event.release.tag_name }}
```

Then this runs as a job in the `PostRelease` workflow together with building the other containers.

Reply:
Sorry, do I need the secrets? This doesn't push to DockerHub so I don't think I need the username or token. This only pushes assets to the release. I had a lot of troubles with the `workflow_call` so I'd rather keep this as-is. Essentially, it wasn't clear to me what was going on and going back and testing it again takes a decent amount of time that I don't really have

Reply:
No you're right you don't need the secrets